packages/components/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Fix to avoid duplicated Y-axis ticks when the Y max value was 0.
 - Remove decimals from Y-axis when displaying currencies.
 - Fix date formatting on charts in Safari.
+- Make `href`/linking optional in SummaryNumber.
```

Comment:
I just published `1.4.0` yesterday so this will need to be under a new "unreleased" heading in this file for `1.4.1`

Reply:
Fixed. thanks!